c/tskit/tables.h

```diff
@@ -2765,13 +2767,14 @@ they appear in the ``nodes`` argument. Specifically, this subsets and reorders
 each of the tables as follows (but see options, below):
 
 1. Nodes: if in the list of nodes, and in the order provided.
-2. Individuals and Populations: if referred to by a retained node, and in the
-   order first seen when traversing the list of retained nodes.
-3. Edges: if both parent and child are retained nodes.
-4. Mutations: if the mutation's node is a retained node.
-5. Sites: if any mutations remain at the site after removing mutations.
-
-Retained edges, mutations, and sites appear in the same
+2. Individuals: if referred to by a retained node.
```

Comment:
We should clarify somewhere that its *just* the individuals referred to by nodes that are retained. We don't keep the rest of the pedigree reachable from those individuals (right?)

Reply:
Yeah, that'd be more like `simplify`.